app/models/events/steps/personal_details.rb

```diff
@@ -7,10 +7,18 @@ class PersonalDetails < ::GITWizard::Step
       attribute :first_name
       attribute :last_name
       attribute :is_walk_in, :boolean
+      attribute :accepted_policy_id
+      attribute :event_id
 
       validates :email, presence: true, email_format: true
       validates :first_name, presence: true, length: { maximum: 256 }
       validates :last_name, presence: true, length: { maximum: 256 }
+      validates :event_id, presence: true
+      validates :accepted_policy_id, presence: true
+
+      def latest_privacy_policy
```

Comment:
I've seem this method in a few places now. Maybe we can add it to a model concern and include it ?

Reply:
Yeah that's a good shout; there's more we could DRY up here as well; the first step of all the sign up journeys always ask for the same attributes (first name, last name and email) as well as containing the privacy policy logic. All of this could be shifted into the `dfe_wizard` gem and shared between sign up journeys in the TTA and GiT apps. I've created a separate ticket for this as its a reasonably big change in itself so I'd prefer to pick it up separately:

https://trello.com/c/kc0Bl99o/3972-make-a-generic-identifystep-model-in-wizard-to-share-logic-across-git-tta